src/databricks/labs/ucx/source_code/path_lookup.py

```diff
@@ -87,9 +87,6 @@ def _is_egg_folder(path: Path) -> bool:
             and any(subfolder.name.lower() == "egg-info" for subfolder in path.iterdir())
         )
 
-    def has_path(self, path: Path):
-        return next(p for p in self._sys_paths if path == p) is not None
-
```

Comment:
This code assumed that `next()` would return `None` if no path matched, but it would instead raise `StopIteration`.

This method is not used anywhere, so I removed it.